presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/cli/TestRun.java

```diff
@@ -246,7 +246,7 @@ private Environment getEnvironment()
                     exposePort(container, 5007); // debug port
                 }
 
-                if (Boolean.getBoolean("CONTINUOUS_INTEGRATION")) {
+                if (Boolean.parseBoolean(System.getenv("CONTINUOUS_INTEGRATION"))) {
```

Comment:
`FlakyTestRetryAnalyzer` kicks in when `CONTINUOUS_INTEGRATION` env is set (regardless of value), so this can be

```suggestion
                if (System.getenv("CONTINUOUS_INTEGRATION") != null) {
```